chart/templates/configmap.yaml

```
@@ -16,11 +16,9 @@ data:
       metricsSink: {{ .Values.controller.metricsSink | quote }}
       tracer:
         sink: {{ .Values.controller.tracer.sink | quote }}
-        enable: {{ .Values.controller.tracer.enable }}
         sampleRate: {{ .Values.controller.tracer.sampleRate }}
       profiler:
         sink: {{ .Values.controller.profiler.sink | quote }}
```

Hence I would follow the metrics approach as a single parameter is availble
```suggestion
      profilerSink: {{ .Values.controller.profilerSink | quote }}
```

---

sounds reasonable